src/pages/build/links-paths-and-anchors.md

```
@@ -290,21 +290,21 @@ let links_to_all_movies = all_first_letter_paths
 
-* [`hdi::prelude::hdk_link_types`](https://docs.rs/hdi/latest/hdi/prelude/attr.hdk_link_types.html)
-* [`hdk::prelude::create_link`](https://docs.rs/hdk/latest/hdk/link/fn.create_link.html)
-* [`hdk::prelude::delete_link`](https://docs.rs/hdk/latest/hdk/link/fn.delete_link.html)
+* [`hdi::link::hdk_link_types`](https://docs.rs/hdi/latest/hdi/link/attr.hdk_link_types.html)
+* [`hdk::link::create_link`](https://docs.rs/hdk/latest/hdk/link/fn.create_link.html)
+* [`hdk::link::delete_link`](https://docs.rs/hdk/latest/hdk/link/fn.delete_link.html)
 * Getting hashes from data
-    * [`hdk::prelude::Record#action_address`](https://docs.rs/hdk/latest/hdk/prelude/struct.Record.html#method.action_address)
-    * [`hdk::prelude::HasHash<T>`](https://docs.rs/hdk/latest/hdk/prelude/trait.HasHash.html)
-    * [`hdk::prelude::Action`](https://docs.rs/hdk/latest/hdk/prelude/enum.Action.html) (contains fields with hashes of referenced data in them)
+    * [`holochain_integrity_types::record::Record#action_address`](https://docs.rs/holochain_integrity_types/latest/holochain_integrity_types/record/struct.Record.html#method.action_address)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)